src/Commands/FlexDayCommand.php

```diff
@@ -4,9 +4,9 @@
 
```

Comment:
I want to confirm if the signature name is ok? @drewroberts

Reply:
yes, that's good for me.